lib/logger.js

```diff
@@ -384,19 +384,35 @@ const flushAll = function(cb) {
     if (!cb || typeof cb !== 'function') {
         cb = debug;
     }
+
+    // Variables used to wait for all the flushes to be completed
+    let counter = 0;
+    const loggersCount = loggers.length;
+    const complete = (errors) => {
+        if (errors.length > 0) {
+            return cb(`The following errors happened while flushing all loggers: ${errors}`);
+        }
+        return cb();
+
+    };
+
     const errors = [];
     loggers.forEach((logger) => {
         logger._flush((err) => {
             if (err) {
```

Comment:
use `error` for readability since the array is `errors` - basically, adding `error` into `errors`: the list of `error`s

Reply:
I don't think it makes a difference.